frontend/src/components/regions/CardDetails/ReviewPopUp.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+
+import Markdown from "react-markdown";
+import Modal from "../../widgets/Modal";
+import Card from "@material-ui/core/Card"
+import CloseIcon from "@material-ui/icons/Close"
+import { Button, CardContent, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
+  cardStyle: {
+    diplay: "block",
+    height: "90vh",
+    width: "90vw",
+    overflow: "scroll",
+    position: "relative",
+    maxWidth: "90%"
+  },
+  cardContentStyle: {
+    maxWidth: "90%",
+  },
+  exitIcon: {
+    position: "absolute",
+    top: "5px",
+    right: "1px"
+  },
+  reviewerEmailStyle: {
+    fontSize: "100%",
+    fontWeight: "bold",
+  }
+}))
+
+const ReviewPopUp = ({ review, openReviewPopUp, setOpenReviewPopUp }) => {
+
+  const classes = useStyles()
+  const closeModal = () => setOpenReviewPopUp(false)
+
+  return (
+    <Modal
+      open={openReviewPopUp}
+      onClose={closeModal}
+    >
+      <Card
+        className={classes.cardStyle}
+      >
+        <CardContent
+          className={classes.cardContentStyle}
+        >
+          <div>
+            <Button
+              className={classes.exitIcon}
+              onClick={closeModal}>
+              <CloseIcon fontSize="small" />
+            </Button>
+            <Typography
+              className={classes.reviewerEmailStyle}
+            >
+              {review.reviewerUserEmail}'s review:⤵

```

Comment:
@sheenarbw , I think just displaying the review is enough here.`airtable` makes good use of presenting just the text in the modal, because that is what the modal is for(I also think me adding the reviewer email breaks the purpose of the modal). Also, the only reason a student usually gets a long review is when the project is not yet competent. 

I think the modal should just have the review and the exit icon only

Reply:
Airtable doesn't know what else would be relevent when opening up a text box. If I am reading a review I shouldn't need to close the review details if I want to see who it is from. 

Reviews are more like emails than just a textfield. If I am reading an email then I like to know who sent it, when, and what was the subject line.  If I have one view for seeing that and then another view for seeing the contents then that would be jarring.